WebRoot/skins/_base/base4/skin.properties

```diff
@@ -987,26 +987,26 @@ PropertySheetNoLeftSpacing  = padding-left:0px;
 ################
 #   SASH - splits a pane
 ################
-SashSize                    = 5px
-SashPadding                 = 1px
-Sash                        =
-SashContainer               = height:100%;
+SashSize                    = 1px
+SashPadding                 = 0px
+Sash                        = background:@SeparatorColor@; position:relative;
+SashContainer               = width:100%; height:100%; position:absolute;
 SashThumb                   = left:50%; position:absolute; top:50%;
 
 ###################
 #   VERTICAL SASH - splits a pane into two wide panels
 ###################
 VSashSize                   = height:@SashSize@; padding:@SashPadding@ 0px; width:100%;
-VSash                       = @VSashSize@
-VSashContainer              = @RowResizeCursor@
+VSash                       = @VSashSize@ @Sash@
+VSashContainer              = @RowResizeCursor@; height:10px;
 VSashThumb                  = margin-left:-7px; margin-top:-1px;
 
 ################
 #   HORIZONTAL SASH - splits a pane into two tall panels
 ################
 HSashSize                   = height:100%; padding:0px @SashPadding@; width:@SashSize@;
-HSash                       = @HSashSize@
-HSashContainer              = @ColResizeCursor@
+HSash                       = @HSashSize@ @Sash@
+HSashContainer              = @ColResizeCursor@; width:10px;
 HSashThumb                  = margin-left:-1px; margin-top:-7px;
```

Comment:
If we are not showing sash thumb image then please remove commented code from
templates/dwt/Widgets.template
dwt.Widgets#DwtVerticalSash
dwt.Widgets#DwtHorizontalSash

and also remove images from img directory, if not used anywhere

this is not important from functionality point of view but will have to have clean code for better understanding